base/shell/explorer/CMakeLists.txt

```diff
@@ -13,7 +13,6 @@ list(APPEND SOURCE
     settings.cpp
     shellservice.cpp
     startctxmnu.cpp
-    startmnu.cpp
```

Comment:
I would suggest you revert this change -- as well as the change of the CreateStartMenu() function -- to keep the PR simple enough. (Also there is no reason why to move the code of this file to somewhere else. You've put the start menu creation code into tray window file, which doesn't make sense from the point of view of code separation)

Reply:
Reverted.